cluster/aws.go

```diff
@@ -230,50 +246,48 @@ func getStateStoreForCluster(clusterType *model.ClusterModel) (stateStore state.
 	return stateStore
 }
 
-// GetKubicornProfile creates *cluster.Cluster from ClusterModel struct
-func GetKubicornProfile(cs *model.ClusterModel) *kcluster.Cluster {
-	uuidSuffix := uuid.TimeOrderedUUID()
-	return &kcluster.Cluster{
-		Name:     cs.Name,
-		Cloud:    kcluster.CloudAmazon,
-		Location: cs.Location,
-		SSH: &kcluster.SSH{
-			Name:          cs.Name + "-" + uuidSuffix,
-			PublicKeyPath: "~/.ssh/id_rsa.pub",
-			User:          "ubuntu",
-		},
-		KubernetesAPI: &kcluster.KubernetesAPI{
-			Port: "443",
-		},
-		Network: &kcluster.Network{
-			Type:       kcluster.NetworkTypePublic,
-			CIDR:       "10.0.0.0/16",
-			InternetGW: &kcluster.InternetGW{},
-		},
-		Values: &kcluster.Values{
-			ItemMap: map[string]string{
-				"INJECTEDTOKEN": kubeadm.GetRandomToken(),
-			},
+func GetMasterServerPool(cs *model.ClusterModel, nodeServerPool []*kcluster.ServerPool, uuidSuffix string) *kcluster.ServerPool {
+	var ingressRules = make([]*kcluster.IngressRule, 2+len(nodeServerPool))
```

Comment:
Why don't you use the append function here for the slice? It will remove the 2+ ...

Reply:
fixed